.github/workflows/ci.yml

```diff
@@ -102,6 +102,12 @@ jobs:
         uses: buildpacks/github-actions/setup-pack@v5.7.2
       - name: Pull builder image
         run: docker pull ${{ env.INTEGRATION_TEST_CNB_BUILDER }}
+      - name: Check for tests that start the container
```

Comment:
The tests that start the container don't actually do any image pulling. The app image will have already been built at that point, by the `pack build` run by any integration test (whether it uses `start_container` or not).

ie: The run image should always be pulled for any integration test, rather than only some of them.

Reply:
Ah, okay. This I wasn't sure about. Thanks for the clarification.

Reply:
Dropped this check